src/main/controllers/GeneralCorrespondenceNotificationDetailsController.ts

```diff
@@ -17,6 +17,7 @@ export default class GeneralCorrespondenceNotificationDetailsController {
   public get = async (req: AppRequest, res: Response): Promise<void> => {
     const userCase = req.session.userCase;
     const selectedCorrespondence = userCase.sendNotificationCollection.find(it => it.id === req.params.itemId);
+    userCase.selectedRequestOrOrder = selectedCorrespondence;
```

Comment:
Not for this PR to sort, but I think we need to move away from this pattern of setting data. We had a similar situation with other notification types where the "view" page sets this data and the "reply" page expects to find it meaning we can't redirect to the reply page before viewing the view page.

I recognise that this is a bigger refactor and will need discussing with the team, just wanted to note it here so it doesn't get lost